index.html

```diff
@@ -316,7 +316,7 @@ <h3>Normative Specifications</h3>
                     <dt id="integrity" class="spec">Linked Data Integrity (LDI)</dt>
                     <dd>
                         <p>
-                            This specification defines a generic framework for expressing proofs of integrity of Linked Datasets via, for example, digital signatures, proofs of work, or proofs of existence. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as well as a means to express these proofs in an RDF Dataset in a way that enables a 3rd party to verify the integrity of the data.
+                            This specification defines a framework for expressing proofs of integrity of RDF Datasets in an RDF Graph. The group defines that framework to work with RDH, although the hashing algorithm, and other constituents of proofs of integrity, are identified as assertions, allowing the same framework to be used with other algorithms. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as a prominent approach used as a proof of integrity. The specification enables a 3rd party to verify the integrity of the data.
```

Comment:
```suggestion
                            This specification defines a framework for expressing, in an RDF Graph, proofs of integrity of RDF Datasets. The group defines that framework to work with RDH, although the hashing algorithm, and other constituents of proofs of integrity, are identified as assertions, allowing the same framework to be used with other algorithms. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as a prominent approach used as a proof of integrity. The specification enables a 3rd party to verify the integrity of the data.
```
I find it clearer like that

Reply:
Agreed